engine/collection/ingest/engine.go

```diff
@@ -203,13 +204,16 @@ func (e *Engine) onTransaction(originID flow.Identifier, tx *flow.TransactionBod
 		return fmt.Errorf("node is not assigned to any cluster in this epoch: %d", counter)
 	}
 
+	localClusterFingerPrint := localCluster.Fingerprint()
+	txClusterFingerPrint := txCluster.Fingerprint()
+
 	log = log.With().
-		Hex("local_cluster", logging.ID(localCluster.Fingerprint())).
-		Hex("tx_cluster", logging.ID(txCluster.Fingerprint())).
+		Hex("local_cluster", logging.ID(localClusterFingerPrint)).
```

Comment:
is fingerprint an ID? I understand that this function, AFAIR just hexprints it, but seems semantically wrong to use `ID()` on `fingerprint`

Reply:
I think fingerprint is different than ID here as it ignores some data parts I guess, but since I don't know that much about this part of the code, I won't change it right now.

Reply:
I see you're point on semantics and the issue is we don't have a fingerprint type, and it still returns an Identifier type